vpr/src/pack/cluster.cpp

```diff
@@ -108,7 +120,11 @@ static bool is_atom_blk_in_pb(const AtomBlockId blk_id, const t_pb* pb);
 static void add_molecule_to_pb_stats_candidates(t_pack_molecule* molecule,
                                                 std::map<AtomBlockId, float>& gain,
                                                 t_pb* pb,
-                                                int max_queue_size);
+                                                int max_queue_size,
+                                                AttractionInfo& attraction_groups);
+
+static void remove_molecule_from_pb_stats_candidates(t_pack_molecule* molecule,
```

Comment:
put a comment on this saying what it does. Removes the molecules from pb_stats->candidates, which removes it from our list of molecules that are candidates to pack for the cluster

Reply:
can be useful for removing blocks that are repeatedly failing. if a block has been found to be illegal, we don't repeatedly consider it